packages/vm/src/runTx.ts

```diff
@@ -1,6 +1,6 @@
-import { Block } from '@ethereumjs/block'
```

Comment:
Ok, I was puzzled for quite some time here when looking at the `runTx()` implementation/changes and wondering why this is already working within a testnet context: so a 4844 tx can *still*  include executable bytecode, is this correct? 🤔 Broader question: has this a practical reason? Or was it also discussed in between (along EIP evolution) if there should be only the blobs in the tx and that's it (no execution at all)?

(my level of L2 interaction understanding is very limited though TBH)

Reply:
Side note: I just also checked `runBlock()` for eventual changes needed but didn't find anything.

Reply:
I honestly haven't heard any discussion on this point but I imagine you want executable bytecode so that you could do state transition updates on an L2 bridge contract to update balances for accounts on the L2 and the executable bytecode would call the new precompile to validate that the data within the blob corresponding to the versionedHash is valid and then pass that proof to the bridge contract and it would update the balances for tokens locked in the bridge

Reply:
But the associated bytecode *does not* have any access to the associated tx blobs along execution, right? 🤔 (e.g. within the precompile)

Would this be a useful thing? Just wondering a bit about the design.

Reply:
It doesn't have access to the blob data but if you pass in the necessary bytes (the inputs to the precompile) in the calldata as part of some contract interaction that validates the precompile, you can do some sort of onchain actions based on verifying that you have some kzg commitments and a proof that are derived from the corresponding blob associated with the transaction.  Something like that anyway.  I haven't seen any concrete implementations to know yet.